docs/collector.mssql.md

```diff
@@ -24,80 +24,80 @@ Name | Description | Type | Labels
-`wmi_mssql_accessmethods_page_deallocations` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_page_allocations` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_page_compressions` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_page_splits` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_probe_scans` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_range_scans` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_scan_point_revalidations` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_ghost_record_skips` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_table_lock_escalations` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_leaf_page_cookie_uses` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_tree_page_cookie_uses` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_workfile_creates` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_worktables_creates` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_accessmethods_worktables_from_cache_ratio` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_availreplica_received_from_replica_bytes` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_sent_to_replica_bytes` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_sent_to_transport_bytes` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_initiated_flow_controls` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_flow_control_wait_seconds` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_receives_from_replica` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_resent_messages` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_sends_to_replica` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_sends_to_transport` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_bufman_background_writer_pages` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_buffer_cache_hit_ratio` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_checkpoint_pages` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_database_pages` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_extension_allocated_pages` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_extension_free_pages` | _Not yet documented_ | counter | `instance`
+`wmi_mssql_accessmethods_au_batch_cleanups` | The number of batches per second that were completed successfully by the background task that cleans up deferred dropped allocation units | counter | `instance`

```

Comment:
We expose this as a counter, rather than a rate, so should be
```suggestion
`wmi_mssql_accessmethods_au_batch_cleanups` | The total number of batches that were completed successfully by the background task that cleans up deferred dropped allocation units | counter | `instance`
```